src/utils/url.ts

```diff
@@ -11,22 +15,11 @@ export const venueInsideUrl = (venueId: string) => {
   return `/in/${venueId}`;
 };
 
-const createAdminUrlHelperFor = (segment: string) => (
-  venueId?: string,
-  selectedTab?: string
-) =>
-  segment
-    ? generatePath(`/admin-ng/:segment?/:venueId?/:selectedTab?`, {
-        segment: segment,
-        venueId: venueId,
-        selectedTab: selectedTab,
-      })
-    : generatePath("/admin-ng");
-
-export const adminNGRootUrl = createAdminUrlHelperFor("");
-export const adminNGVenueUrl = createAdminUrlHelperFor("venue");
-export const adminNGSettingsUrl = createAdminUrlHelperFor("advanced-settings");
-export const ADMIN_CREATE_SPACE_URL = "/admin-ng/create/venue";
+export const adminNGVenueUrl = (venueId?: string, selectedTab?: string) =>
+  generatePath(ADMIN_V3_VENUE_URL, { venueId, selectedTab });
+
+export const adminNGSettingsUrl = (venueId?: string, selectedTab?: string) =>
+  generatePath(ADMIN_V3_ADVANCED_URL, { venueId, selectedTab });
```

Comment:
What happens when `selectedTab` is undefined? Is it going to fallback to 0 and show the first tab?

Reply:
The same that happens when `/admin/path/id/segment` is replaced with `/admin/path/id` in URL, it's up to the routed component to determine what to do with a missing URL param